experimental/opencv-text-recognition/text_recognition.py

```diff
@@ -11,9 +11,15 @@
 from tqdm import tqdm
 from scipy.ndimage import interpolation as inter
 import cv2
+from typing import List
+
+
+def correct_skew(image: np.ndarray, delta=0.05, limit=5) -> np.ndarray:
+
```

Comment:
parameters with default values should also be typed, e.g. `correct_skew(delta: float=0.05, limit: int=5)`